### DISCUSSION TITLE
OPDS is not syncing read status back to Komga – should it?

### DISCUSSION THREAD

**Discussion by denkristoffer** [gotson/komga#1756] [Q&A] [answered]
Hi 👋 I'm trying to use my Komga server for an OPDS client. It generally works well, but it seems like read status is not being synced back from the client to Komga. Is this simply the way the spec works at the moment with each client maintaining its own status, or is there a way to make the server and client stay in sync?

**Comment by gotson** [comment, marked as answer]
Hi, OPDS v1 does not have support for read progress sync.

OPDS v2 has some unofficial support for it, but it is only implemented in Cantook.

**Reply by denkristoffer** [reply]
> Hi, OPDS v1 does not have support for read progress sync.
> 
> OPDS v2 has some unofficial support for it, but it is only implemented in Cantook.

All right, I see. Thanks for the answer 🙂 

For anyone interested I also found this discussion from the Panels app, where the devs say the same thing, and that they implemented it custom for Komga: https://community.panels.app/t/tracking-progress-in-opds-stream-library/1610/20
